src/conky.cc

```diff
@@ -160,6 +160,7 @@
 
 #ifdef BUILD_BUILTIN_CONFIG
 #include "defconfig.h"
+#include "themespresetmanager.h"
```

Comment:
Once you added a guard like `BUILD_IMLIB2`, the include needs to be guarded with `ifdef`s, as well as any code that's using that.